src/Dhall/Core.hs

```diff
@@ -627,7 +638,7 @@ prettyChunks (Chunks a b) =
             (foldMap prettyMultilineChunk a <> prettyMultilineBuilder b)
         <>  "''"
         )
-    else "\"" <> foldMap prettyChunk a <> prettyText b <> "\""
+    else Pretty.annotate Literal ("\"" <> foldMap prettyChunk a <> prettyText b <> "\"")
```

Comment:
Do you plan to also annotate multi-line string literals?

Reply:
Yep, just haven't got that far.